hpat/tests/test_series.py

```diff
@@ -1485,6 +1485,30 @@ def test_impl():
         hpat_func = hpat.jit(test_impl)
         np.testing.assert_array_equal(hpat_func(), test_impl())
 
+    def test_series_iterator_int(self):
+        def test_impl():
+            A = pd.Series([1, 2, 3, 4, 5])
```

Comment:
What do you think about constructing Series with unsorted list?

Reply:
@akharche Should we add tests with other types, like float and boolean as well? Will they be handled correctly now?

Reply:
@densmirn I think, it doesn't matter in this case

Reply:
@kozlov-alexey Yes, I am going to add by the next commit